data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg

```diff
@@ -63,29 +64,29 @@
 
         {STORY_PART_TALLIN (_ "Well, suppose we do join up with the elves — assuming they will let us — and we make a combined raid on the fortress, what will the orcs most likely do when the ransom slips from their grasp?")}
 
-        {STORY_PART_HAMEL (_ "Kenning the way of orcs, belike they’d kill the Princess, especially if it seems the rescue ha’ any chance of succeeding.")}
+        {STORY_PART_HAMEL (_ "Kenning the way of orcs, they’d kill the princess, especially if it seems the rescue ha’ any chance of succeeding.")}
 
-        {STORY_PART_TALLIN (_ "Exactly. On the other hand, if we raided the fortress while the orcs were talking ransom with the elves, the orcs might hesitate just long enough.")}
+        {STORY_PART_TALLIN (_ "Exactly. On the other hand, if we raided the fortress while the orcs were talking ransom with the elves, the orcs might hesitate just long enough for us to free her.")}
 
         {STORY_PART_HAMEL (_ "Hmmm... I think you’ve the right of it, lad. Most orcs being thicker than a wood-knot, I’d lay odds on Rakshas giving strict orders that she not be touched without his direct word on the matter.")}
 
         {STORY_PART_TALLIN (_ "But it would be unwise to trust that the orcs will stay stupid forever. If we do go ahead with this raid, we are going to have to be in and out like lightning.")}
+         #  po: My shinsplitters are invaluable for this kind of job. They are braw warriors and don't stop fighting until either they or their opponet lie dead. No fortress can stand against them.
+        {STORY_PART_STALRAG (_ "My Shinsplitters be invaluable for this kind o’ job. They be braw warriors and dinna stop their assaulut ‘til either they or their opponent lie dae. Nae a fortress ken stand against them.")}
```

Comment:
Spelling mistake on 'assaulut'. Also, 'till' is the correct abbreviation for 'until' - I don't think it conveys anything in terms of accent to use the more informal ''til'.

Reply:
Did it.